internal/test_cases/type_of_command_test_case.go

```diff
@@ -0,0 +1,76 @@
+package test_cases
+
+import (
+	"fmt"
+	"os/exec"
+	"regexp"
+
+	"github.com/codecrafters-io/shell-tester/internal/logged_shell_asserter"
+	"github.com/codecrafters-io/shell-tester/internal/shell_executable"
+	"github.com/codecrafters-io/tester-utils/logger"
+)
+
+type TypeOfCommandTestCase struct {
+	Command        string
+	SuccessMessage string
```

Comment:
@andy1li if we're allowing `SuccessMessage` to be passed in, how come only `RunForBuiltin` uses it, but the other functions (`RunForExecutable` and the other one) don't? Is there a reason for this?

Reply:
@rohitpaulk Do we want other functions to support custom SuccessMessage as well?

Stage 9 is the only place currently needing a different SuccessMessage:

<img width="679" alt="image" src="https://github.com/user-attachments/assets/c2ae01cb-8763-408e-8465-621ef93082a8" />

Reply:
Hmm, this doesn't seem intentional - we usually stay consistent with all invocations. How bout changing this to always say `✔️ Received "<expectedOutput>"`?